[BUG] test_delta_scan_split_with_DV_enabled_with_DVs failed assert 2 == 1

**Describe the bug**
Build: rapids_it-arm64-pre_release/310

```
[2026-03-31T16:28:01.920Z] FAILED ../../src/main/python/delta_lake_test.py::test_delta_scan_split_with_DV_enabled_with_DVs[DATAGEN_SEED=1774972618, TZ=UTC, INJECT_OOM, ALLOW_NON_GPU(DeserializeToObjectExec,ShuffleExchangeExec,FileSourceScanExec,FilterExec,MapPartitionsExec,MapElementsExec,ObjectHashAggregateExec,ProjectExec,SerializeFromObjectExec,SortExec)] 
- AssertionError: Expected 1 partitions for split read

[2026-03-31T16:28:01.920Z] assert 2 == 1

[2026-03-31T16:28:01.920Z] FAILED ../../src/main/python/delta_lake_test.py::test_delta_scan_split_with_DV_disabled_with_DVs[DATAGEN_SEED=1774972618, TZ=UTC, ALLOW_NON_GPU(DeserializeToObjectExec,ShuffleExchangeExec,FileSourceScanExec,FilterExec,MapPartitionsExec,MapElementsExec,ObjectHashAggregateExec,ProjectExec,SerializeFromObjectExec,SortExec)] 
- AssertionError: Expected 1 partitions for split read

[2026-03-31T16:28:01.920Z] assert 2 == 1
```

**Environment details**
- Pipeline: rapids_it-arm64-pre_release (ARM64 integration tests)
- Spark versions tested: 3.5.6 (cuda13-arm64)
- Branch: release/26.04 (updated from release/26.02 per code changes)
- Recent commit: Add IcebergProvider$ to dist/unshimmed-from-each-spark3xx.txt to fix classloader issue (#14494)

--- a/integration_tests/src/main/python/delta_lake_test.py
+++ b/integration_tests/src/main/python/delta_lake_test.py
@@ -375,7 +375,7 @@ def test_delta_scan_split_with_DV_disabled_with_DVs(spark_tmp_path, pushdown_dv_
         assert num_deleted > 0, "Expected some rows to be deleted"
         spark.sql(f"ALTER TABLE delta.`{data_path}` SET TBLPROPERTIES " +
                   "('delta.enableDeletionVectors' = 'false')")
-    # The cuDF-based reader (GpuDeltaParquetFileFormat2), which is used when dv_predicate_pushdown is True, support the file split,
+    # The cuDF-based reader (GpuDeltaParquetFileFormat2), which is used when dv_predicate_pushdown is True, supports the file split,
     # whereas the scala reader (GpuDeltaParquetFileFormat) does not support it.
     # So we expect 2 partitions when dv_predicate_pushdown is True, and 1 partition when it is False.
     expected_num_partitions = 2 if pushdown_dv_predicate else 1

